src/main/java/com/google/devtools/build/lib/bazel/BazelWorkspaceStatusModule.java

```diff
@@ -182,6 +192,7 @@ public ActionResult execute(ActionExecutionContext actionExecutionContext)
       stableMap.put(BuildInfo.BUILD_USER, username);
       volatileMap.put(
           BuildInfo.BUILD_TIMESTAMP, Long.toString(getCurrentTimeMillis(clientEnv) / 1000));
+      volatileMap.put("FORMATTED_DATE", format(getCurrentTimeMillis(clientEnv) / 1000 * 1000));
```

Comment:
I understand that is just a cherrypick of a submitted change, but this looks wrong to me.

If the `clientEnv` does not have `SOURCE_DATE_EPOCH` we can end up outputting a `BUILD_TIMESTAMP` and `FORMATTED_DATE` that do not match.

Reply:
Right, nice catch! I am preparing a fix.

Reply:
PTAL